Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builder image
-FROM golang:1.12-alpine as builder
+FROM golang:1.13-alpine as builder
```

Comment:
Do we need to change https://github.com/pingcap/tidb/blob/release-2.1/circle.yml#L6?